packages/neuron-ui/.storybook/main.ts

```diff
@@ -1,21 +1,21 @@
 export default {
-  "stories": ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
-  "addons": [
-    "@storybook/addon-links",
-    "@storybook/addon-essentials",
-    "@storybook/addon-interactions",
-    "@storybook/preset-create-react-app",
-    "storybook-addon-react-router-v6"
-  ],
-  "framework": "@storybook/react",
-  core: {
-    builder: "webpack5"
+  stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
+  addons: ["@storybook/addon-links", "@storybook/addon-essentials", "@storybook/addon-interactions", "@storybook/preset-create-react-app", "storybook-addon-react-router-v6"],
+  framework: {
+    name: "@storybook/react-webpack5",
+    options: {}
   },
   webpackFinal: config => {
     config.resolve.alias = {
       ...config.resolve.alias,
       electron: require.resolve('./electron')
-    }
-    return config
+    };
```

Comment:
Semicolons should be avoided.

Ref: https://github.com/nervosnetwork/neuron/blob/develop/packages/neuron-ui/.prettierrc.js#L6